third_party/move/move-model/src/options.rs

```diff
@@ -11,6 +12,9 @@ pub struct ModelBuilderOptions {
     /// part of the model.
     pub compile_via_model: bool,
 
+    /// The language version to use.
+    pub language_version: LanguageVersion,
```

Comment:
nit: `pub language_version_to_use: LanguageVersion`; and no comment

Reply:
Same here.